README.md

```diff
@@ -11,7 +11,7 @@ This list contains only stage 1 proposals and higher that have not yet been with
 
 | 🚀 | Proposal                                                                                                    | Champion                           | Stage |
 |---|-------------------------------------------------------------------------------------------------------------|------------------------------------|-------|
-|   | [SIMD.JS - SIMD APIs](https://github.com/tc39/ecmascript_simd/)                                             | John McCutchan, Peter Jensen, Dan Gohman, Daniel Ehrenberg | 3 |
+|   | [SIMD.JS - SIMD APIs](https://github.com/tc39/ecmascript_simd/)                                             | John McCutchan, Peter Jensen, Dan Gohman, Daniel Ehrenberg | 0? |
```

Comment:
This proposal should be removed from this table completely. Also you should add it to https://github.com/tc39/proposals/blob/master/inactive-proposals.md